diff --git a/node_modules/vue-router/types/vue.d.ts b/node_modules/vue-router/types/vue.d.ts
index 307515c..9148f26 100644
--- a/node_modules/vue-router/types/vue.d.ts
+++ b/node_modules/vue-router/types/vue.d.ts
@@ -7,7 +7,6 @@ import VueRouter, { Route, RawLocation, NavigationGuard } from './index'
 
 declare module 'vue/types/vue' {
   interface Vue {
-    $router: VueRouter
     $route: Route
   }
 }
